mandatory/2-function-creation.js

```diff
@@ -5,27 +5,37 @@ Write a function that:
 - removes any forward slashes (/) in the strings
 - makes the string all lowercase
 */
-function tidyUpString(strArr) {}
+function tidyUpString(strArr) {
```

Comment:
Great job Daniel, I really like your approach to this particular section and how you've simplified it.